localstack/services/awslambda/lambda_executors.py

```diff
@@ -1190,6 +1190,17 @@ def reset_default_env_variables(env_vars_before):
             if env_value_before is None:
                 os.environ.pop(env_name, None)
 
+    def execute_go_lambda(self, event, context, main_file, func_details=None):
+        event_json_string = "%s" % (json.dumps(event) if event else "{}")
+        cmd = "AWS_LAMBDA_FUNCTION_HANDLER=%s AWS_LAMBDA_EVENT_BODY='%s' %s" % (
```

Comment:
@pinzon We probably want to pass in the event body as an environment variables dict, rather than in serialized string form (to avoid escaping issues). Not critical for this PR, though - we can pick this up in a follow-up change..